include/networkit/io/NetworkitBinaryGraph.hpp

```diff
@@ -23,10 +23,8 @@ enum WEIGHT_FORMAT {
 enum MASKS {
```

Comment:
Please define `MASKS` and `WEIGHT_FORMAT` as `enum class` reduce pollution of the namespace.

Reply:
An `enum class` does not work well here: we need the numerical values of those constants (and enum classes need casts to access that). But you are right that this pollutes the namespace. My suggestion: move this to a `namespace nkbg` and change the values to be `static constexpr uint8_t DELETED_BIT = 1;` and similar (with a constant-specific type instead of `uint8_t`). This also has the advantage that they will have the proper type.